app/services/cart_items/fetch.rb

```diff
@@ -0,0 +1,13 @@
+module Shop
+  module Services
+    module CartItems
+      class Fetch
+        def call(product_id:)
+          cart_item = CART_ITEMS.find { |cart_item| cart_item.product.id == product_id }
```

Comment:
```
app/services/cart_items/fetch.rb:6:42: W: Shadowing outer local variable - cart_item.
          cart_item = CART_ITEMS.find { |cart_item| cart_item.product.id == product_id }
                                         ^^^^^^^^^
```

You used the same variable name for local variable and variable inside block - it isn't good